app/Http/Controllers/TrainingDash.php

```diff
@@ -802,6 +777,17 @@ public function saveNewTrainerFeedback(Request $request) {
         $redirect = ($request->input('redirect_to') == 'internal') ? '/dashboard' : '/';
         return redirect($redirect)->with('success', 'Thank you for the feedback! It has been received successfully.');
     }
+
+    public function handleSchedule() {
+        $user = Auth::user();
+
+        if ($user->rating_id == 1 && !$user->onboarding_complete) {
+            return redirect()->back()->with('error', 'ZTL Onboarding must be complete before scheduling a training session. Please refer to the onboarding course on VATUSA before continuing. Contact the TA at ta@ztlartcc.org with questions or concerns.');
```

Comment:
Change error text to read: "Onboarding must be complete before scheduling a training session. Please refer to the ZTL onboarding course on the VATUSA Academy. Contact the TA with questions or concerns."

Reply:
Resolved and squashed into 76ccc76